dev/oracle/oldest/docker-compose.yml

```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved
+
+services:
+  oracle-oldest:
+    container_name: sequelize-oracle-oldest
+    image: gvenzl/oracle-xe:18-slim
+    environment:
+      ORACLE_PASSWORD: password
```

Comment:
Do we not need to provide a `ORACLE_DATABASE` for this version? We provide it for 23.4